src/platforms/x86_64_linux_musl.rs

```diff
@@ -221,8 +236,10 @@ fn build_crt(
                 "-DCOMPILER_RT_BUILD_MEMPROF='Off'",
                 "-DCOMPILER_RT_BUILD_ORC='Off'",
             ])
-            .args(crate::platforms::SHARED_BUILD_OPTS)
```

Comment:
Per discussion, I worry about `229: "-DLLVM_ENABLE_ASSERTIONS='Off'"` in some builds, as otherwise-unsafe code in LLVM is guarded by assertions.